docs/getting_started.rst

```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
 Initialize a Logging Session
 ########################
 
-TBD
+An example script for creating a logging session can be found `here <https://whylogs.readthedocs.io/en/latest/auto_examples/log_dataframe.html#script>`__.
 
 Create a Logger
 ########################
 
-TBD
+Loggers log statistical information about your data. They have the following parameters:
+
+- **dataset_name** sets the name of the dataset, to be used in DatasetProfile metadata and generated filenames.
+- **dataset_timestamp** sets start and end timestamps for the data.
```

Comment:
this just means "the timestamp" that the data represents (ether the time of the batch or the current data window)